writing-visualising/dashboards_rshiny.qmd

```diff
@@ -7,36 +7,148 @@ title: "Creating R Shiny dashboards"
 
+For some initial inspiration, check out the [Shiny gallery](https://shiny.posit.co/r/gallery/).
+
+### Getting started with Shiny
+
 There are a lot of resources already available to support you when working with R Shiny, if you're new, then this [blog post](https://www.r-bloggers.com/2021/03/a-gentle-introduction-to-r-shiny/) provides a gentle introduction.
 
+If you like learning by reading/doing then the [Posit Shiny-specific site](https://shiny.posit.co/r/getstarted/shiny-basics/lesson1/index.html) has lots of resources that you can  work through. Likewise [Introduction section of Mastering Shiny](https://mastering-shiny.org/basic-intro.html) is also a helpful resources.
+
+If you like learning by watching then the [Posit Youtube channel](https://www.youtube.com/@PositPBC/videos) is also full of good resources. 
+
+### Creating a user interface
+
 Our [template shiny app repository](https://github.com/dfe-analytical-services/shiny-template) is a useful starting point for all public facing dashboards in the department. Please see the [dashboard template section](#dfe-dashboard-template) for guidance on what the dashboard template includes and how to use it.
 
-For more advanced shiny knowledge it's worth taking a look at the guide to [engineering production-grade shiny apps](https://engineering-shiny.org/).
+[Shiny layouts](https://shiny.posit.co/r/layouts/) gives a high level idea of how to structure an R Shiny app. Alongside this it is recommended you take a look at [`bslib`](https://rstudio.github.io/bslib/index.html) which makes constructing beautiful user interface easy. This site is full of information on how to construct the page and how to work themeing. If you like watching it done then [this video](https://www.youtube.com/watch?v=3l3w2p1Fy4M) is a good place to start.
+
+:::{.callout-warning}
+Previously a lot of dashboards used `shinydashboard` - if starting afresh we **recommend `bslib` instead**. `shinydashboard` is uses an old version of bootstrap and has not been updated in many years. It shouldn't immediately break but a risk nonetheless.
+:::
+
+### R Shiny Best Practice
+
+Some key things you should include when building an app:
+
+- Your code should follow the [DfE Quality Assurance Framework](https://educationgovuk.sharepoint.com/sites/sarpi/g/SitePages/Quality-Assurance.aspx) 
+- You need to [**Accessibility Test**](#accessibility-testing-in-r-shiny) regardless of internal or external deployment
+- Your code should be [tested](#code-testing)
+- Your file structure should be [well-organised](https://mastering-shiny.org/best-practices.html#code-organization)
+- You should use [dependency management](https://mastering-shiny.org/best-practices.html#dependency-management)
+- If using [databases](#databases-with-shiny), you should use the pool package and ensure you are careful of SQL Injection
```

Comment:
Not sure if we have much info on SQL injection - worth adding an explainer sentence here maybe so people know what it is? Either here or in the actual section itself